tutorials/graph-hello-world/graph-hello-world.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Hello Graph! Write Your First SAP Graph Application
+description: Get a brief introduction to SAP Graph, including the SAP Graph Navigator and programming basics.
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, software-product>sap-business-technology-platform]
+primary_tag: software-product>sap-integration-suite
+---
+
+## Details
+### You will learn
+  - What SAP Graph entails
+  - How to navigate connected data entities in the SAP Graph Navigator
+  - `hello world` programming essentials to try out SAP Graph
+
+  SAP Graph is a new unified API for SAP. It addresses run-away API complexity and integration challenges by providing developers a single connected and unified view of all their business data, consolidating the data models of data sources like SAP S/4HANA, SAP Sales Cloud and SAP SuccessFactors into one, unified and connected, data model, representing all the data in a landscape. We call this the Business Data Graph.
+
+  !![SAP Graph Overview](SAP-Graph-overview_sm.png)
+
+  A data graph is a connected graph, with data entities serving as the nodes of the graph and edges that represent relationships (which we call associations) between those entities. The nodes of the Business Data Graph are the common enterprise business objects, like Customer, Supplier, Product, Sales and Purchase Order, General Ledger, etc.
+
+SAP Graph was productively released for early adopters in December of 2021. We offer dedicated support for early adopters of this innovative solution. [Sign up to be an early adopter] (https://influence.sap.com/sap/ino/#campaign/2940) of SAP Graph or contact us at sap.graph@sap.com if you are interested.
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Explore SAP Graph on the SAP Graph Navigator)]
+
```

Comment:
Please remove this line!

Reply:
Not entirely clear what is meant here, but I removed the empty line preceding the step-begin.